pages/common/csvpy.md

```diff
@@ -0,0 +1,12 @@
+# csvpy
+
+> Loads a CSV file into a Python shell.
```

Comment:
I'll be honest, I still don't get it even after reading @rubenvereecken's explanation. Maybe I should try it out :P